### PR TITLE
build/ops: rpm: move ceph-*-tool binaries out of ceph-test subpackage

### PR DESCRIPTION
**Pull request by smithfarm**
ceph-osdomap-tool into ceph-osd subpackage
ceph-monstore-tool into ceph-mon subpackage
ceph-kvstore-tool into ceph-base

Fixes: http://tracker.ceph.com/issues/21762
Signed-off-by: Nathan Cutler <ncutler@suse.com>

Thanks to @vumrao for the proposal, to @ktdreyer for the suggestion to put ceph-kvstore-tool in ceph-base, and to @tchaikov for bringing the ceph-kvstore-tool manpage to my attention!

### CODE REVIEW

**Comment by smithfarm** [comment]
Since ceph-osd and ceph-mon are typically installed on all test nodes, I don't think this will break teuthology. . .

**Comment by smithfarm** [comment]
Updated, and deb added.

File: debian/ceph-base.install [hunk 1]
```diff
@@ -5,6 +5,7 @@ usr/bin/ceph-run
 usr/bin/crushtool
 usr/bin/monmaptool
 usr/bin/osdmaptool
+usr/bin/ceph-kvstore-tool
```
Comment (tchaikov) [inline debian/ceph-base.install]: @smithfarm could you please take this chance to add `usr/share/man/man8/ceph-kvstore-tool.8` to this list?

File: ceph.spec.in [hunk 2]
```diff
@@ -1632,9 +1635,6 @@ fi
 %{_bindir}/ceph_tpbench
 %{_bindir}/ceph_xattr_bench
 %{_bindir}/ceph-coverage
-%{_bindir}/ceph-monstore-tool
-%{_bindir}/ceph-osdomap-tool
-%{_bindir}/ceph-kvstore-tool
 %{_bindir}/ceph-debugpack
 %{_mandir}/man8/ceph-debugpack.8*
 %{_mandir}/man8/ceph-kvstore-tool.8*
```
Comment (tchaikov) [inline ceph.spec.in]: move this line to the section of `%files base`.

**Comment by smithfarm** [comment]
@tchaikov Updated and added you to credits in the PR description

File: debian/ceph-test.install [hunk 3]
```diff
@@ -1,8 +1,5 @@
 usr/bin/ceph-client-debug
 usr/bin/ceph-coverage
-usr/bin/ceph-kvstore-tool
```
Comment (tchaikov) [inline debian/ceph-test.install]:
```
2017-10-16T15:02:39.462 INFO:teuthology.orchestra.run.smithi112.stdout:dpkg: error processing archive /var/cache/apt/archives/ceph-test_13.0.0-2017-g207de03-1xenial_amd64.deb (--unpack):
2017-10-16T15:02:39.463 INFO:teuthology.orchestra.run.smithi112.stdout: trying to overwrite '/usr/share/man/man8/ceph-kvstore-tool.8.gz', which is also in package ceph-base 13.0.0-2017-g207de03-1xenial
```
see http://qa-proxy.ceph.com/teuthology/kchai-2017-10-16_14:55:45-rados-wip-kefu-testing-2017-10-16-1746-distro-basic-smithi/1739413/teuthology.log

weird ...

File: debian/ceph-test.install [hunk 4]
```diff
@@ -28,4 +25,3 @@ usr/bin/ceph_tpbench
 usr/bin/ceph_xattr_bench
 usr/lib/ceph/ceph-monstore-update-crush.sh
 usr/share/java/libcephfs-test.jar
-usr/share/man/man8/ceph-kvstore-tool.8
```
Comment (smithfarm) [inline debian/ceph-test.install]: @tchaikov I forgot to drop this line - fixed

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2017-10-17_17:30:15-rados-wip-kefu-testing-2017-10-17-2334-distro-basic-smithi/
- http://pulpito.ceph.com/kchai-2017-10-18_01:59:18-rados-wip-kefu-testing-2017-10-17-2334-distro-basic-mira/

**Pull request merged**